editoast/src/views/v2/train_schedule.rs

```diff
@@ -682,41 +683,104 @@ pub async fn project_path(
 #[derive(Debug, Serialize, ToSchema)]
 enum SimulationSummaryResultResponse {
     // Minimal information on a simulation's result
-    #[allow(dead_code)]
     Success {
         // Length of a path
         length: u64,
         // travel time
-        time: f64,
+        time: u64,
         energy_consumption: f64,
     },
-    // Pathfinding not found for a train id
-    #[allow(dead_code)]
-    PathfindingNotFound,
-    // Running time not found or a train id
-    #[allow(dead_code)]
-    RunningTimeNotFound,
+    // Pathfinding fails for a train id
+    PathfindingFailed,
+    // Running time fails or a train id
+    RunningTimeFailed,
 }
 
 /// Retrieve simulation information for a given train list.
 /// Useful for finding out whether pathfinding/simulation was successful.
 #[utoipa::path(
     tag = "train_schedulev2",
+    params(SimulationBatchParams),
     responses(
-        (status = 200, description = "Project Path Output", body = HashMap<Identifier, SimulationSummaryResultResponse>),
+        (status = 200, description = "Project Path Output", body = HashMap<i64, SimulationSummaryResultResponse>),
     ),
 )]
 #[get("/simulation_summary")]
 pub async fn simulations_summary(
```

Comment:
Why is this function called `simulations_summary` (with a plural form for simulations) whereas the endpoint is `simulation_summary` (singular form)?